src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a contact in contact list
+ */
+public class Contact {
+    protected boolean isPrivate;
+
+    public boolean isPrivate() {
+        return isPrivate;
+    }
+}
```

Comment:
You can consider `@Override` both `equals()` and `hashcode()` methods for `Contact` class as well. Perhaps you want to find out why we may want to override these two methods.